integration-tests/server/test/manageSets.js

```diff
@@ -1,27 +1,33 @@
 import { expect } from 'chai';
-import gql from 'graphql-tag';
 import { print } from 'graphql';
+import gql from 'graphql-tag';
 
 export default ({ api, graphqlField, gqlPath }) => {
   let setId = undefined;
+
   it('creates set successfully', async () => {
-    let response = await api.post({
-      endpoint: gqlPath,
-      body: {
-        query: print(gql`
+    const { data } = await api
```

Comment:
api's default fetcher now gives with full response object (data, status, etc.) to allow custom error handling while integrating... this change accounts for that.